skopt/tests/test_utils.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+import tempfile
+
+from sklearn.utils.testing import assert_array_almost_equal
+from sklearn.utils.testing import assert_array_equal
+from sklearn.utils.testing import assert_array_less
+from sklearn.utils.testing import assert_equal
+from sklearn.utils.testing import assert_not_equal
+from sklearn.utils.testing import assert_less_equal
+from sklearn.utils.testing import assert_greater
+from sklearn.utils.testing import assert_true
```

Comment:
could you remove the imports that aren't used please? Needed: assert_true, assert_equal and assert_array_equal